packages/office-ui-fabric-react/src/components/GroupedList/GroupedListSection.tsx

```diff
@@ -123,12 +124,16 @@ export class GroupedListSection extends BaseComponent<IGroupedListSectionProps,
 
     const { selection, group } = props;
 
+    initializeComponentRef(this);
+
     this._id = getId('GroupedListSection');
 
     this.state = {
       isDropping: false,
       isSelected: selection && group ? selection.isRangeSelected(group.startIndex, group.count) : false
     };
+
+    this._events = new EventGroup(this);
```

Comment:
Same here re: `EventGroup` vs. `on` as the latter would require a larger refactor.